data_science/datasciencetopics.md

```diff
@@ -0,0 +1,65 @@
+# Data Science Topics and Subtopics
+
+### Terminal
+- Directories
+- Terminal commands
+- Installation and setup
+- Jupyter notebook
+
+### Git and GitHub
+- Cloning Repositories
+- Version Control
+- Branching, pull requests, working with a team
+
+### Markdown
+- Intro to basic syntax (i.e. headings, lists, code blocks, embedding links and images)
+
+### Progrmming Concepts
+- Iteration
+- Conditional Statements
+- Sequences (Array, List)
+- Data Types (Strings, numbers, logical)
+
+### Python 1 - Intro to Python
+- Integrating programming concepts
+- Dictionaries
+- Lists
+- Tables
+- Operational and Logical Operands
+
+### Python 2 - Functions
+- Writing functions
+- Try-catch
+
+### Pandas and Seaborn

```

Comment:
I really don't think pandas should be linked with seaborn, I think seaborn makes more sense to be under `Data Visualization and Data Analysis`, but data analysis could be moved to pandas, i.e. `Pandas and Data Analysis` and `Seaborn and Data Visualization` maybe?

Or maybe seaborn and pandas should each have their own section.

Some topics for pandas could be regarding to method chaining, series, windowing/aggregating (grouping, i.e. `group_by`)?

Reply:
Every time I've worked with data wrangling and analysis, I have had to work primarily with NumPy. Since Pandas and Seaborn are put in as their own sections, would it be useful to to have NumPy as one of these sections too? Although NumPy also would not be necessary if it was simply in the Pandas and Seaborn sections, since both libraries use NumPy, and it is generally pretty simple to use so creating a separate section for it might also be too much.